src/services/DfusionService.ts

```diff
@@ -146,7 +146,7 @@ export class DfusionRepoImpl implements DfusionService {
 
   public watchOrderPlacement(params: WatchOrderPlacementParams) {
     this._contract.events
-      .OrderPlacement()
+      .OrderPlacement({ fromBlock: 6360000 }) // allows to get a bunch of past orders at once
```

Comment:
I think you need to remove this before you merge, right?

Reply:
:grin: [I remember](https://github.com/gnosis/dex-telegram/pull/192#discussion_r412030278)